.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,18 @@
+name: Release-drafter
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
Can you confirm we need this trigger on pull_request? I don't the requirement on the official docs or general usage as well.

Reply:
We normally don't do push on master branch, but I added this just in case anybody having authorization to push on master and really doing it.

Reply:
We do push on master branch.

Reply:
Do we need it on every push or pull? Don't we need this only on the release or release candidate tag?

Reply:
> Do we need it on every push or pull? Don't we need this only on the release or release candidate tag?

@vrozov Line #6 and #7 just intends to log the changes pushed on master. If we don't have to track those changes, I'll remove this.

Reply:
This is github workflow/action definition file and it defines when action is triggered. I don't think it is necessary to trigger release draft action on every PR or push to the master. IMO, that should be merged into CD workflow.

Reply:
It'll be triggered on the PRs only when they get merged to master branch, not all PRs. I thought that release notes intend to track all changes merged/pushed to master branch(Please correct me if I'm wrong), which is why the action gets triggered up on PR merge and commit push on master branch.

Reply:
I do see it running on your PR that suggests that it will be triggered on every PR and every PR merge. Unless I miss something, I don't see why this is necessary and why it can't run only as part of CD.

Reply:
There was a miscommunication. I should say that the release note gets updated only when PRs get merged to master branch or commits get pushed on master branch. The action runs on other PRs, but it stops nearly immediately without updating release notes at all.

Reply:
It is not necessary to update release notes each time PR is merged. It is also not necessary to run the action on PR update (whether it exits without updating release notes or not).

Reply:
I've removed the trigger for PR updates. But I believe that the action should be triggerd when PRs get merged to master. Otherwise, it wouldn't update release notes with the merge PR.

Reply:
`pull_request` event is triggered when pull request is created against a branch and not when it is merged. You will have to specifically check for pull request `closed` and `merged` events but not needed here